cmd/osbuild-tests/main_test.go

```diff
@@ -81,10 +81,14 @@ func TestComposeCommands(t *testing.T) {
 	uuid = buildCompose(t, "empty", "qcow2")
 	defer deleteCompose(t, uuid)
 
-	// https://github.com/osbuild/osbuild-composer/pull/180
-	// uuid = startCompose(t, "empty", "tar")
-	// time.Sleep(time.Second)
-	// runComposer(t, "compose", "cancel", uuid.String())
+	// workers ask the composer every 15 seconds if a compose was canceled.
+	// Use 20 seconds here to be sure this is hit.
+	uuid = startCompose(t, "empty", "tar")
+	defer deleteCompose(t, uuid)
```

Comment:
Why do you have to start another compose while a few lines above we've got one already started ?

Reply:
It had a comment saying that it tests reusing the cache. Not sure how it actually does that, but I thought adding one that is immediately canceled wouldn't do that.

Reply:
IDK what the cache is but reordering the actions like currently seems to be better.